gulpfile.js

```diff
@@ -193,7 +196,7 @@ gulp.task('serve', function() {
     //not sure what, if any, benefit there would be from doing it one way or the other...
 
     //enable live reloading by watching for trigger filetypes (html, css, js, etc) and notifying the server
-    gulp.watch(['dist/client/**/*.html', 'dist/client/**/*.css', 'dist/client/**/*.js'], ['livereload']);
+    gulp.watch(['dist/client/**/*.html', 'dist/client/**/*.css', 'dist/client/**/*.js', 'dist/server/*.js'], ['livereload']);
```

Comment:
same here with the glob syntax. Add the double asterisk indicate all subdirectories.